src/net_processing.cpp

```diff
@@ -2785,7 +2782,6 @@ void PeerLogicValidation::ProcessMessage(
         if (!pfrom.fInbound) {
             // Mark this node as currently connected, so we update its timestamp later.
             LOCK(cs_main);
```

Comment:
hm, cs_main also removed in that backport and one more line above

Reply:
Resolved in latest push